slack/Package.md

```diff
@@ -1,221 +1,21 @@
-[![Build](https://github.com/ballerina-platform/module-ballerinax-slack/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-slack/actions?query=workflow%3ACI)
-[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-slack.svg)](https://github.com/ballerina-platform/module-ballerinax-slack/commits/master)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+Connects to Slack from Ballerina
 
-# Ballerina Slack Connector 
-Connects to Slack using Ballerina
+## Package Overview
+This is a [Ballerina](https://ballerina.io/) connector for Slack.
 
-# Introduction
-## Slack
-[Slack](https://api.slack.com/) is a channel-based messaging platform. With Slack, people can work together more effectively, connect all their software tools and services, and find the information they need to do their best work — all within a secure, enterprise-grade environment.
+This package allows you to access the Slack Web API and Slack Events API through Ballerina.
+It provides the capability to query information from and perform some actions in a Slack workspace. It also allows you to listen to Slack Events.
```

Comment:
query information and
again, we need to mention what the some operations are.

Reply:
Fixed the first issue.

Second one -> The number of operations we support is a big list. Those details are available in the api docs. We don't need to repeat it here. In Choreo, those operations will be shown in a dropdown list. Users can refer those places.

If we add them here, in the future when we add new operations, we'll have to edit here also. So better not to mention them here.